### PR TITLE
feat(print): add basic support for printing pages

### PR DESCRIPTION
**Pull request by lionelB**
add print support

### CODE REVIEW

File: packages/code-du-travail-frontend/static/print.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+body {
+  font-size: 10pt;
+}
+header.section-white {
```
Comment (douglasduteil) [inline packages/code-du-travail-frontend/static/print.css]: can't be replaced by a *BEM-ed* class like ".main-header" ?

**Review by douglasduteil — CHANGES_REQUESTED** [review]
Wow... :doughnut: 
Can you *sort* the classes by contextual use ?
I like the Inverted Triangle CSS (itcss) architecture.
Can you at least group the classes by *reach* ?

```css

/**
 * 
 * Generic
 * =======
 * 
 * Ground-zero styles (Normalize.css, resets, box-sizing).
 * 
 */

/**
 * 
 * Base
 * ========
 * 
 * Unclassed HTML elements (type selectors)
 * 
 */
 

/**
 * 
 * Objects
 * =======
 * 
 * Cosmetic-free design patterns.
 * 
 */

/**
 * 
 * Components
 * ==========
 * 
 * Designed components, chunks of UI.
 * 
 */

/**
 * 
 * Trumps
 * ======
 * 
 * Helpers and overrides
 * 
 */
```

**Comment by revolunet** [comment]
👍

**Comment by lionelB** [comment]
@douglasduteil  thanks for review ! Didn't know that much itcss, does it seems better like that ?

**Comment by douglasduteil** [comment]
Not really :sweat_smile: 
Check this out https://www.youtube.com/watch?v=1OKZOV-iLj4 
(especially at 34:19 https://youtu.be/1OKZOV-iLj4?t=2059)

**Review by douglasduteil — APPROVED** [review]
Alright. It's better yeah

**Pull request merged**